programs/debug_node/README.md

```diff
@@ -1,104 +0,0 @@
-
-Introduction
-------------
```

Comment:
Pushed to bitshares-core.wiki: https://github.com/bitshares/bitshares-core/wiki/README-debug_node